behat.yml.dist

```diff
@@ -15,13 +15,15 @@ default:
         - SendInterviewContext
         - EventContext
         - TalkContext
+        - SpeakerContext
   extensions:
     Behat\Symfony2Extension:
       kernel:
         bootstrap: features/bootstrap/bootstrap.php
         class: App\Kernel
     Behat\MinkExtension:
         base_url: http://nginx
+        files_path: base_url"/public/images/"
```

Comment:
The previous comment here said : `a filesystem path, absolute or relative`. Here's an attempt to use an http address again, and I do believe it is not interpreted the way you think it is, so your string is actually just the plain `base_url"/public/images/"`.
Also, this highlights the fact the tests are not good enough, as this value is wrong but there is no failing tests...